typescript/common-resolvers/src/types.ts

```diff
@@ -162,10 +162,20 @@ export type Repository = {
   };
   label: {
     add: Add<DbLabelCreateInput>;
+    addMany: (
+      args: {
+        labels: DbLabelCreateInput[];
+        imageId: string;
```

Comment:
Again, I don't think it makes sense to have `imageId` here, it restricts it to only one image when it should be more generic, like we could use it to import all labels from a COCO json import in one single query instead of image by image.

Reply:
Let's leave it like for the moment as it fulfills our needs. If you want to adapt it for the COCO labels import, we can create an other issue for that

Reply:
I agree with @valerian with the `imageId`. Please make `addMany` directly reuse the `imageId` and `labelClassId` of the `Label` entity as it should have been with such a generic function.